demo/src/main/java/com/example/demo/controller/FhirController.java

```diff
@@ -0,0 +1,82 @@
+package com.example.demo.controller;
+
+import org.hl7.fhir.r4.model.Patient;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import ca.uhn.fhir.context.FhirContext;
+import ca.uhn.fhir.parser.IParser;
+
+import com.example.demo.service.ProprietaryApiService;
+
+import java.util.stream.Collectors;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+@RestController // Kennzeichnet diese Klasse als Spring REST Controller
+@RequestMapping("/fhir") // Basis-URL für alle Endpunkte in dieser Klasse
+public class FhirController {
+
+    private static final Logger logger = Logger.getLogger(FhirController.class.getName());
+
+    @Autowired
+    private ProprietaryApiService proprietaryApiService;
+
+    // Erzeugt ein FhirContext-Objekt für FHIR R4
+    private final FhirContext fhirContext = FhirContext.forR4();
+
+    @PostMapping("/Person") // Mapped HTTP POST-Anfragen auf diesen Endpunkt
+    public ResponseEntity<String> createPatient(@RequestBody String patientResource) {
+        try {
+            // Loggt die erhaltene Anfrage
+            logger.info("Received request to create patient: " + patientResource);

```

Comment:
Es steht zwar nicht in den aktuellen Anforderungen, aber ich denke nicht, dass wir Patientendaten in die Logdatei schreiben sollten.